server/views/eligibility/exceptionalCircumstances.pug

```diff
@@ -0,0 +1,36 @@
+extends ../layout
+
+block content
+
+    - var exclusionValue = data.decision || null;
+
+    div.pure-g
+
+        div.pure-u-1
+            include ../includes/back
+            include ../includes/personalDetailsSummary
+            include ../includes/errorBanner
+
+        div.pure-u-1-2
+            h2.heading-large Have exceptional circumstances been accepted?
+
+            if errorObject.decision
+                p.error-message Please select an answer
+
+            form(method="post")
+                input(type="hidden" name="_csrf" value=csrfToken)
+                input(type="hidden" name="nomisId" value=nomisId || '')
+
+                div.largePaddingBottom
+                    div.form-group.inline.smallPaddingTop.largePaddingBottom
+                        .multiple-choice
+                            input#ec1(type='radio', checked=exclusionValue === 'Yes' name='decision', value='Yes')
+                            label(for='ec1') Yes
```

Comment:
"exclusionValue" - exclusion is something else. this one should be named exceptionalCircumstancesValue

Also, the ID 'ec1' isn't clear - try exceptionalCircumstancesYes etc